cluster-scope/base/cert-manager.io/issuers/ingress-letsencrypt-production/issuer.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ingress-letsencrypt-production
+  namespace: openshift-ingress
+spec:
+  acme:
+    email: ops-team@operate-first.cloud
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: letsencrypt-production-key
+    solvers:
+      - selector:
+          dnsZones:
+            - "na.operate-first.cloud"
+        dns01:
+          cnameStrategy: Follow
+        cloudDNS:
+          project: aicoe-prow
+          hostedZoneName: 'na-operate-first-cloud'
+          serviceAccountSecretRef:
```

Comment:
Doesn't this SA need to be created? @larsks ?

Reply:
@HumairAK I'm not even sure what the `serviceAccountSecretRef` is for; on the MOC `ocp-prod` cluster we're using AWS, and all we needed was a `secretRef` with the AWS credentials. I'll have to look at the cert-manager docs.

Reply:
Touching base -- needed to create iam service account via gcloud, we did that and I added the resulting SA secret..